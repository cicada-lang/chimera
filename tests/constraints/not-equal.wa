find q {
  1 != 1
}

find q {
  1 != 2
}
