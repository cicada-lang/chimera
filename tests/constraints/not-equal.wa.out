[]
[ _.0 ]
[]
[]
[ [3, 5] ]
[ ["cat", _.0] with { disj { _.0 != 3 } } ]
[ _.0 with { disj { _.0 != 1 } } ]
[ [_.0, _.1] with { disj { _.0 != 3 } } ]
